src/group_theory/order_of_element.lean

```diff
@@ -655,18 +655,41 @@ begin
   exact mem_powers_iff_mem_range_order_of
 end
 
-lemma order_eq_card_powers [decidable_eq α] {a : α} :
-  order_of a = fintype.card (submonoid.powers a : set α) :=
+/-- The equivalence between `fin (order_of a)` and `submonoid.powers a`, sending `i` to `a ^ i`. -/
+noncomputable def fin_equiv_powers {a : α} :
```

Comment:
Since the `order_of_element` file is now generalised to additive groups, could you add an additive version of the new lemmas?